plugins/node.d/amavis

```diff
@@ -1,4 +1,6 @@
 #!/bin/sh
+# -*- sh -*-
+# vim: ft=sh
```

Comment:
I share your editor preference, but I do not think, that we need to add these markers to the plugin files. Instead it can be done via private configuration.